client/src/components/Questions.tsx

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import {ThemeProvider, theme, CSSReset, Flex, Box, Image, Heading, HStack, Stack, Button} from "@chakra-ui/react";
+import ReactDOM from 'react-dom';
+
+export const Questions = () => {
+    return (
+        <ThemeProvider theme={theme}>
+            <CSSReset />
+            <QuestionsPage />
+        </ThemeProvider>
+    );
+}
+
+const QuestionsPage = () => {
+    return (
+        <Flex minHeight='100vh' width='full' align='center' justifyContent='center'>
+            <Box backgroundImage="url('/src/components/Sausage.png')" borderWidth={1} px={1} width='full' maxWidth='500px' borderRadius={4} textAlign='center' boxShadow='lg'>
+                <Image src='./Sausage.png'  />
+                <ThemeProvider theme={theme} />
+                <Box p={2}>
+                    <QuestionHeader />
+                    <QuestionTitle />
+                </Box>
+                <Box p={3}>
+                    <Stack isInline  spacing='70px' justify='center'>
+                        <Box borderWidth = "3px" borderColor = "#000000" as="button" borderRadius="0px" bg="orange" color="black" px={4} h='100px' w='200px'>
+                            Answer 1
```

Comment:
Instead of this Answer 1 text can you pass an answer in as a parameter? You should pass from Questions to QuestionsPage a dictionary object with some data like
`{
    questionTitle: "why does this happen?",
    answers: {
        answer1: "Answer 1",
        answer2: "Answer 2",
    }
}`

Reply:
Okay noted. I will do so